CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the "vscode-didact" extension will be documented in this
 
 ## 0.1.9
 
+ - Expanded autocompletion to cover Didact URIs in AsciiDoc documents
  - Adding autocompletion to ease creation of Didact URIs in Markdown documents
```

Comment:
from a user point of view we can combine the 2 as it goes in same release.

So maybe just replace with "- Adding autocompletion to ease creation of Didact URIs"